Assignment1_q2_Node.py

```diff
@@ -0,0 +1,16 @@
+# Node is a unit which is used in my implementation of Singly Linked List
+class Node(object):
```

Comment:
Since `Node` is part of the linked list data structure, it could be defined in the same module (i.e., in the same file).